pandas/tests/groupby/test_quantile.py

```diff
@@ -248,6 +248,33 @@ def test_groupby_quantile_skips_invalid_dtype(q):
     tm.assert_frame_equal(result, expected)
 
 
+def test_groupby_quantile_NA_float(any_float_dtype):
+    # GH#42849
+    df = DataFrame({"x": [1, 1], "y": [0.2, np.nan]}, dtype=any_float_dtype)
+    result = df.groupby("x")["y"].quantile(0.5)
```

Comment:
can you do a case with a listlike qs e.g. [0.5, 0.75]

Reply:
added below